client/connection/openvpn.go

```diff
@@ -34,7 +40,8 @@ func ConfigureVpnClientFactory(
 		signer := signerFactory(consumerID)
 
 		statsSaver := bytescount.NewSessionStatsSaver(statsKeeper)
-		statsSender := bytescount.NewSessionStatsSender(mysteriumAPIClient, vpnSession.ID, providerID, signer)
+		clientCountry, _ := detectCountry(ipResolver, locationDetector)
```

Comment:
Don't swallow error

Reply:
I would not throw error also. Lets implement logging. So that, client knows what is happening

Reply:
made error checking